otter/convergence/steps.py

```diff
@@ -222,4 +226,47 @@ def as_effect(self):
         Produce a :obj:`Effect` to remove some nodes from some RCv3 load
         balancers.
         """
-        return _rackconnect_bulk_request(self.lb_node_pairs, "DELETE", (204,))
+        eff = _rackconnect_bulk_request(self.lb_node_pairs, "DELETE",
+                                        success_pred=has_code(204, 409))
+        # While 409 isn't success, that has to be introspected by
+        # _rcv3_check_bulk_delete in order to recover from it.
+        return eff.on(_rcv3_check_bulk_delete)
+
+
+_UUID4_REGEX = ("[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}"
+                "-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}")
+_RCV3_NODE_NOT_A_MEMBER_PATTERN = re.compile(
+    "Node (?P<node_id>{uuid}) is not a member of Load Balancer Pool "
+    "(?P<lb_id>{uuid})".format(uuid=_UUID4_REGEX),
+    re.IGNORECASE)
+
+
+def _rcv3_check_bulk_delete(result):
+    """
+    Checks if the RCv3 bulk deletion command was successful.
+
+    The request is considered successful if the response code
+    indicated unambiguous success, or the machine we're trying to
+    remove is already removed anyway.
+
+    If the machine was already removed, returns the next step to try
+    and remove the remaining pairs, unless a fatal error *also*
+    occurred. This is necessary because RCv3 bulk requests are
+    atomic-ish.
+    """
+    response, body = result
+
+    if response.code == 204:  # All done!
+        return
+
+    pairs_to_delete = []
+    for error in body["errors"]:
+        match = _RCV3_NODE_NOT_A_MEMBER_PATTERN.match(error)
```

Comment:
First, a question - do we actually want to wedge a group if the load balancer is not active or has been deleted?  In the case of the load balancer being deleted, it seems like our task may be complete (no load balancer, we can go ahead and delete the servers).  The way we handle CLBs currently is that if, during removing a server from a load balancer, we find out that the load balancer has been deleted we count that as a success.

Secondly, bulk delete can bulk delete from any number of load balancers.  This doesn't address the case of if there are operations across different RCv3 load balancers for a particular group.  It just bails as soon as a single non-matching pattern is found, whereas the deletes for the other load balancer may be doable. 

A couple examples of multiple load balancer per group may be (1) if the user just has 2 RCv3 load balancers for their group, or (2) if the user has several generations of servers, each on an different RCv3 load balancer, that they are now deleting.

The question of what should happen if a load balancer isn't active, or one load balancer may not be active, may be complex, so I'm also ok with filing a ticket about this and handling it later.